commands/init.go

```diff
@@ -44,26 +58,65 @@ func withClient(fn func(c client.Client, cmd *cobra.Command, args []string) erro
 	}
 }
 
-func InitClientWithUsernameAndPassword(username, password, instanceURL string) (*model.Client4, string, error) {
+func isValidChain(chain []*x509.Certificate) bool {
+	// check all certs but the root one
+	certs := chain[:len(chain)-1]
+
+	for _, cert := range certs {
+		if _, ok := insecureSignatureAlgorithms[cert.SignatureAlgorithm]; ok {
+			return false
+		}
+	}
+	return true
+}
+
+func VerifyCertificates(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
```

Comment:
The rawCerts are received here, but not used.

Reply:
They are there to comply with the function signature that `tls.Config.VerifyPeerCertificate` expects

Reply:
Makes sense. Thanks for the explanation.